scripts/histmakers/3dsmoothing/combinedifferentbinning.C

```diff
@@ -0,0 +1,86 @@
+//TO BE USED IN CASE WE NEED TO COMBINE EFFICIENCIES SMOOTHED IN DIFFERENT PT RANGES (AT THE MOMENT THE SMOOTHER DOESN'T DO IT, SO A WORKAROUND IS TO CHANGE THE PT RANGE IN THE HISTOGRAMS AND THEN ADD THE REMAINING BINS FROM ANOTHER FILE).
+
+void combinedifferentbinning() {
```

Comment:
Where is this used? I think this would be much, much simpler using the boost hist python interface

Reply:
This is not used in the analysis, this is used to create the new smoothed efficiencies (which are then fed to WRemnants)